src/client/src/services/model/currencyPairPosition.js

```diff
@@ -1,10 +1,12 @@
 export default class CurrencyPairPosition {
   _symbol:string;
+  _ccy:string;
   _basePnl:number;
   _baseTradedAmount:number;
 
   constructor(symbol:string, basePnl:number, baseTradedAmount:number) {
     this._symbol = symbol;
+    this._ccy = symbol.length === 6 ? symbol.substr(0, 3) : '';
```

Comment:
Also, we aren't running out of characters on our hard drives. :)  can we call this by it's full name?  so either `.currency` or `.baseCurrency`